ceremony.md

```diff
@@ -0,0 +1,16 @@
+# Ceremony
+
+An out-of-band provisioning of cryptographic keys is required to bootstrap cryptographic security among participants of re-vault. Countermeasures should be in place to minimize the risks associated with physical, network, and social engineering attacks that aim to compromise this process. Ideally this ceremony would not be too cumbersome for the humans involved and the scope for human error should be minimized.
+
+## Objectives
+
+Prior to the ceremony, it is assumed that the architectural configuration has been specified, the devices and machines have been aquired and the appropriate software has been installed (including blockchain download for all full nodes). 
```

Comment:
nit: acquired

Reply:
> and the appropriate software has been installed

What precisely need to be installed prior to the ceremony ? On the top of my head i'd say that since they will have to configure xpubs and onions, they'll need a Revault wallet. So they'll need a synced `bitcoind` (i'm assuming this but since they don't need no balance nor transaction i'm not even sure it's required).

Reply:
I suppose it should be specified according to the role of the machine.
**Sync Server**:  sync server app, tor
**Co-signers**: bitcoind (fully synced), co-signer app, tor
**Watchtowers**: bitcoind (fully synced), watchtower app, tor
**Manager PC**: bitcoind (fully synced), revault wallet
**Non-manager PC**: bitcoind (fully synced), revault wallet
**Hardware Security Module**: wallet functionality (including PSBT, and secure PST storage)

What is missing?

Reply:
tor for the wallets too, otherwise looks good.